releasenotes/releasenotes_v5.rst

```diff
@@ -121,6 +135,10 @@ Fix
 Release 5.7.0, May 29th, 2024
 -----------------------------
 
+.. note::
+
+    Due to a security vulnerability we recommend not to use version 5.7.0. Please update to version 5.10.1. See :ref:`vonk_releasenotes_5_10_0` for more information.
```

Comment:
```suggestion
    Due to a security vulnerability we recommend not to use version 5.7.0. Please update to version 5.10.1. See :ref:`vonk_releasenotes_5_10_1` for more information.
```

Reply:
This should point to the hotfix version, no?

Reply:
Good point indeed, I overlooked that.